src/main/scala/pl/mowczarek/love/actors/Field.scala

```diff
@@ -24,11 +33,25 @@ class Field(x: Int, y: Int, gameMap: ActorRef) extends Actor with ActorLogging {
       creatures.filter(_ != sender).foreach { creatureRef =>
         creatureRef forward command
       }
+
+    case command @ Migrate =>
```

Comment:
You don't have to use reference to the case object, since you don't use it in the code below. 'command @ Migrate' should be just Migrate.